string action
skeleton/DetectionMsg[] boundingBoxes
int64 timestamp